### DISCUSSION TITLE
Hiding RelationManager on some pages

### DISCUSSION THREAD

**Discussion by jwhoami** [filamentphp/filament#5627] [Help] [answered]
Hi,

I have role->hasMany->users relationmanager setup and is working fine.

However, I want the relationmanager to show up on the view page and not on the edit page.

Is this possible?

Thanks

**Comment by jwhoami** [comment]
any suggestions?
Thanks

**Comment by danharrin** [comment, marked as answer]
Override `getRelationManagers()` on the edit page to return `[]` and `getRelationManagers()` on the view page to return `[YourRelationManager::class]`

**Reply by jwhoami** [reply]
Thanks @danharrin. works like a charm
